src/hotspot/share/gc/shared/cardTable.cpp

```diff
@@ -262,3 +274,220 @@ void CardTable::print_on(outputStream* st) const {
   st->print_cr("Card table byte_map: [" PTR_FORMAT "," PTR_FORMAT "] _byte_map_base: " PTR_FORMAT,
                p2i(_byte_map), p2i(_byte_map + _byte_map_size), p2i(_byte_map_base));
 }
+
+MemRegion CardTable::committed_for_region_in_shared_virtual_space(const MemRegion mr) const {
```

Comment:
Can you please add some comments to this function? The name is unclear to me.

Reply:
> Can you please add some comments to this function? The name is unclear to me.

I have renamed the method to committed_card_table_mem_for_shared_virtual_space_region, which is clearer